print/components/PicassoPeriod.vue

```diff
@@ -65,8 +88,7 @@ export default {
   },
   computed: {
     periodChunks() {
-      // TODO: choose chunkSize based on orientation
-      const chunkSize = 3
+      const chunkSize = this.landscape ? 5 : 3
```

Comment:
In ecamp v2 it was 8 / 4: https://github.com/ecamp/ecamp/blob/devel/src/application/print/class/build_picasso.php#L73..L76
Is that too crowded? I always felt those were quite sensible defaults.

Reply:
Didn't really give it a lot of thought, to be honest. Can try with 8/4 and see how it looks. Look and feel depends probably a lot on how many parallel events are scheduled.

Reply:
Increased to 7 in landscape and 4 in portrait. 8 in landscape seems to me rather dense.

https://github.com/ecamp/ecamp3/pull/3026/commits/83826a275107797778cf59a84d90fd8cfe19a0f0